sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_internal/algorithms/aes_cbc.py

```diff
@@ -39,7 +39,6 @@ def finalize(self, data):
         padded = self._ctx.finalize()
         return self._padder.update(padded) + self._padder.finalize()
 
-    @property
```

Comment:
Assuming that it is intentional change to remove property flag? these look like private classes only anyway

Reply:
yes